xdsl/traits.py

```diff
@@ -212,21 +212,38 @@ class SymbolOpInterface(OpTrait):
     """
 
     @staticmethod
-    def get_sym_attr_name(op: Operation) -> StringAttr:
+    def get_sym_attr_name(op: Operation) -> StringAttr | None:
         """
         Returns the symbol of the operation
```

Comment:
maybe update the docstring wrt the updated return value (i.e., `None`)

Reply:
Also, the class docstring?

Reply:
Updated, WDYT?